apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleForm.vue

```diff
@@ -0,0 +1,139 @@
+<template>
```

Comment:
Question: How can we take the folder structure of the components used as this form?

Reply:
Yeah.. We need to discuss it. I think we need to change the folder structure soon. 🥲